### ISSUE TITLE
Change category management to load from models rather than default  json file (Trac #535)

### ISSUE THREAD

**Issue by ajj** [SasView/sasview#668]
Probably need to keep the user json file for users preferences. However, initial population of category list should be from models rather than the default_categories.json file.

Migrated from http://trac.sasview.org/ticket/535
```json
{
    "status": "closed",
    "changetime": "2016-10-07T21:11:43",
    "_ts": "2016-10-07 21:11:43.802811+00:00",
    "description": "Probably need to keep the user json file for users preferences. However, initial population of category list should be from models rather than the default_categories.json file.",
    "reporter": "ajj",
    "cc": "",
    "resolution": "fixed",
    "workpackage": "SasView Bug Fixing",
    "time": "2016-03-18T16:05:58",
    "component": "SasView",
    "summary": "Change category management to load from models rather than default  json file",
    "priority": "major",
    "keywords": "",
    "milestone": "SasView 4.1.0",
    "owner": "mathieu",
    "type": "defect"
}
```

**Comment by butlerpd** [comment]
Trac update at `2016/03/20 22:50:53`: **butler** changed milestone from "SasView 4.0.0" to "SasView Next Release +1"

**Comment by butlerpd** [comment]
Trac update at `2016/09/05 03:01:59`: **butler** changed milestone from "SasView Next Release +1" to "SasView 4.1.0"

**Comment by mdoucet** [comment]
Trac update at `2016/10/07 21:11:43`: 
* **mathieu** commented:

In changeset 212bfc297ca102b648c949f030d3d06a4c569fa3:

```text
#!CommitTicketReference repository="sasview" revision="212bfc297ca102b648c949f030d3d06a4c569fa3"
Pull categories from models. Get rid of default categories. Fixes #668
```


* **mathieu** changed owner from "" to "mathieu"
* **mathieu** changed resolution from "" to "fixed"
* **mathieu** changed status from "new" to "closed"
